### PR TITLE
tickets/SITCOM-687

### PR DESCRIPTION
**Pull request by mareuter**

### CODE REVIEW

File: python/lsst/ts/genericcamera/fits_header_items_generator.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def _fixup_value(self, value):
             keyword_value = value.strip().replace("'", "")
         except AttributeError:
             if value is None:
-                keyword_value = ""
+                keyword_value = None
```
Comment (tribeiro) [inline python/lsst/ts/genericcamera/fits_header_items_generator.py:169]: I guess you could simply do:

```
keyword_value = value
```

now

File: doc/version_history.rst [hunk 2]
```diff
@@ -6,6 +6,11 @@
 Version History
 ###############
 
+v1.3.0
```
Comment (tribeiro) [inline doc/version_history.rst:9]: are you sure this needs to be a minor release? I would probably make is a patch instead, e.g. v1.2.2.

**Review by tribeiro — APPROVED** [review]
Looks good, but that a look at the two comments in left in the code.. Also, you may want to update the pre-commit hooks.

File: python/lsst/ts/genericcamera/fits_header_items_generator.py (hunk 1 above)
Comment (mareuter) [reply python/lsst/ts/genericcamera/fits_header_items_generator.py]: Fixed.

File: doc/version_history.rst (hunk 2 above)
Comment (mareuter) [reply doc/version_history.rst]: Agree. Changed it to 1.2.2

**Pull request merged**